crmsh/bootstrap.py

```diff
@@ -1692,6 +1693,106 @@ def init():
     init_network()
 
 
+class JoinLock(object):
+    """
+    Class to manage lock for multiple nodes join in parallel
+    """
+
+    JOIN_LOCK_FILE = "/usr/share/crmsh/.join_lock_file"
+    DETECT_CMD = "test -f {}".format(JOIN_LOCK_FILE)
```

Comment:
Do we need to use this `bash` commands here? 
There are not pythonic options to run this simple operations?

Reply:
I guess that you need them because the commands are executed remotely, but it's quite ugly